test/MudletTest.qmake

```diff
@@ -0,0 +1,83 @@
+######################################################################
```

Comment:
This seems to be autogenerated and should not be checked in.

Reply:
Maybe, will doublecheck.. Obviously this comment line is of no importance.